addon/result_iterator.cc

```diff
@@ -147,18 +147,19 @@ namespace NodeDuckDB {
       case duckdb::LogicalTypeId::INTEGER:
         return  Napi::Number::New(env, val.GetValue<int32_t>());
       case duckdb::LogicalTypeId::BIGINT:
-        #ifdef NAPI_EXPERIMENTAL
-          return  Napi::BigInt::New(env, val.GetValue<int64_t>());
-        #else
-          return  Napi::Number::New(env, val.GetValue<int64_t>());
-        #endif
+        // for now return as string: BigInt is supported by Napi v5+
+        return  Napi::String::New(env, val.ToString());
+      case duckdb::LogicalTypeId::HUGEINT:
+        // for now return as string: BigInt is supported by Napi v5+
+        return  Napi::String::New(env, val.ToString());
       case duckdb::LogicalTypeId::FLOAT:
         return  Napi::Number::New(env, val.GetValue<float>());
       case duckdb::LogicalTypeId::DOUBLE:
         return  Napi::Number::New(env, val.GetValue<double>());
       case duckdb::LogicalTypeId::DECIMAL:
         return  Napi::Number::New(env, val.CastAs(duckdb::LogicalType::DOUBLE).GetValue<double>());
       case duckdb::LogicalTypeId::VARCHAR:
+      case duckdb::LogicalTypeId::BLOB:
         return  Napi::String::New(env, val.GetValue<string>());
```

Comment:
Would `Buffer` be better? That's what we use for binary blobs in Node.

Reply:
I was thinking about TypedArrays (as I wrote in description), but buffers would work as well. let's do that as separate PR (I doubt we return blobs in our queries atm)